core/Flist.cva6

```diff
@@ -89,6 +89,9 @@ ${CVA6_REPO_DIR}/vendor/pulp-platform/fpnew/src/fpu_div_sqrt_mvp/hdl/norm_div_sq
 ${CVA6_REPO_DIR}/vendor/pulp-platform/fpnew/src/fpu_div_sqrt_mvp/hdl/nrbd_nrsc_mvp.sv
 ${CVA6_REPO_DIR}/vendor/pulp-platform/fpnew/src/fpu_div_sqrt_mvp/hdl/preprocess_mvp.sv
 
+//FPGA memories
+${CVA6_REPO_DIR}/common/local/techlib/fpga/rtl/SyncDpRam.sv
```

Comment:
The SyncDpRam.sv is twice in the repository. Why not using the SyncDpRam from the fpga-support submodule ?  The plan is to vendorize the fpga-support repository to allow modifications if needed. Please contact @zchamski to vendorize the submodule if needed.

Reply:
Can this be done in a second time after this PR be merged?

Reply:
I propose a deal: Zbigniew (if he is ok) vendorizes support-fpga to allow you to use the relative files. Fair ? 

In parallel, I merge this PR